include/usb_cam/formats/av_pixel_format_helper.hpp

```diff
@@ -958,6 +960,152 @@ inline AVPixelFormat get_av_pixel_format_from_string(const std::string & str)
   return STR_2_AVPIXFMT.find(fullFmtStr)->second;
 }
 
+/// @brief Get ROS PixelFormat from AVPixelFormat.
+/// @param avPixelFormat AVPixelFormat
+/// @return String specifying the ROS pixel format.
+inline std::string get_ros_pixel_format_from_av_format(const AVPixelFormat & avPixelFormat)
+{
+  std::string ros_format = "";
+
+  switch (avPixelFormat) {
+    default:
+      {
+        ros_format = usb_cam::constants::UNKNOWN;
+      }
+      break;
+
+    case AVPixelFormat::AV_PIX_FMT_RGB24:
+      {
+        ros_format = usb_cam::constants::RGB8;
+      }
+      break;
```

Comment:
@boitumeloruf writing each case like this will make the code much more read-able I think (and a lot less clutter)

```suggestion
    case AVPixelFormat::AV_PIX_FMT_RGB24: ros_format = usb_cam::constants::RGB8; break;
```